docs/tendermint-rpc.md

```diff
@@ -0,0 +1,210 @@
+# Tendermint RPC
+
+[Tendermint RPC](https://docs.tendermint.com/v0.34/rpc/#) can be used to query application's state and/or subscribe to Tendermint WebSocket events.   
+The default RPC listen address is tcp://0.0.0.0:26657. See the [docs](https://docs.tendermint.com/v0.34/rpc/#) to learn how to configure and customize.
+
+## WebSocket Events
+
+Full list of Tendermint WebSocket events can be found [here](https://pkg.go.dev/github.com/tendermint/tendermint/types#pkg-constants).
+
+### Subscribe
+
+Subscribing to Tendermint WebSocket events can be done using any WS clients like [wscat](https://github.com/websockets/wscat), [Postman](https://www.postman.com/), etc. The following is an example of subscribing to a new `tx` (transaction) event using wscat:
+
+1. Connect to the server:
+
+```bash
+wscat -c ws://localhost:26657/websocket
+```
+
+2. Subscribe to the `tx` event in the wscat command line (refer to the [docs](https://docs.tendermint.com/v0.34/rpc/#) to learn the syntax):
+
+```json
+{"jsonrpc":"2.0","method":"subscribe","id":0,"params":{"query":"tm.event='Tx'"}}
+```
+
+You should get the following response if everything is OK:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 0,
+  "result": {}
+}
+```
+
+3. Perform any transaction in another terminal tab to broadcast a `tx` event:
```

Comment:
```suggestion
3. Wait for the transaction

Consider performing any transaction in another terminal tab to broadcast a `tx` event for the test purposes. For example, the following transaction can be sent in case of a local pool (for Test Net or Main Net pools `dcld` needs to be configured properly and write access needs to be given, see https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/docs/how-to.md#cli-configuration)
```